examples/step_bullet_kuka_env.py

```diff
@@ -22,23 +22,18 @@ def step_bullet_kuka_env(n_steps=1000):
 
     """
     # Construct the environment
-    env = BulletEnv(
-        gym.make('KukaBulletEnv-v0',
-                 renders=True,
-                 isDiscrete=True,
-                 maxSteps=10000000))
+    env = BulletEnv(gym.make('KukaBulletEnv-v0', renders=True,
```

Comment:
should people just use `GymEnv` instead? It works, right? Smaller APIs are better.